Update disk quota error message while transfering to atomic

paulopmt1 opened the pull request:
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to p1714397294805299-slack-C03TY6J1A

## Proposed Changes

* We're adding the 95% limit threshold as a warning.

**When can purchase a plan upgrade**:

| Before | After (style fix) |
|--------|--------|
| <img width="700" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/d5739380-fcaf-48f0-b45f-174bf0e2423b"> | <img width="697" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/49c680cd-5a1b-4742-9454-08cc510610ce"> |


**When it's already upgraded**:

| Before | After (copy fix) |
|--------|--------|
|<img width="696" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/c82defb5-480a-4522-8368-1bd6d31394ae"> | <img width="697" alt="image" src="https://github.com/Automattic/wp-calypso/assets/1044309/d57d4e5d-b542-479b-b4ab-db6f856f90a0">|


## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

paulopmt1 committed:
Update disk quota error message while transfering to atomic

paulopmt1 committed:
Improved copy and style

paulopmt1 committed:
Copy improved

RafaelFunchal commented:
LGTM!

Instead of linking to /media, can we link to this public-facing doc?

https://wordpress.com/support/media/#delete-files-from-media

This explanation can save HEs lot of time:

> Deleting a file from Media will also remove it from the post or page it is on. Deleted media files cannot be recovered. You may want to [export a backup of the media library](https://wordpress.com/support/export-your-media-library/) before deleting large amounts of files.

paulopmt1 committed:
Changed the delete media file link to our docs

paulopmt1 commented:
> LGTM!
> 
> Instead of linking to /media, can we link to this public-facing doc?
> 
> https://wordpress.com/support/media/#delete-files-from-media
> 
> This explanation present in the doc can save HEs lot of time:
> 
> > Deleting a file from Media will also remove it from the post or page it is on. Deleted media files cannot be recovered. You may want to [export a backup of the media library](https://wordpress.com/support/export-your-media-library/) before deleting large amounts of files.

That's a good idea @RafaelFunchal. Just updated it.
